### PR TITLE
Add standard defined GQLSTATUS codes for data exceptions

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-201.surge.sh

Updated pages:
- [List of all GQLSTATUS server error codes](https://neo4j-docs-status-codes-201.surge.sh/status-codes/5/errors/gql-errors)

File: modules/ROOT/pages/errors/gql-errors.adoc [hunk 1]
```diff
@@ -100,6 +100,34 @@ Status description:: error: connection exception - no such routing policy. Polic
...
 Some examples are data and constraint creation, which conflicts with existing constraints, properties of non-storable type, and spatial and temporal values with invalid components.
 
+=== 22000
+
+Status description:: error: data exception
+
+=== 22001
+
+Status description:: error: data exception - string data, right truncation
+
```
Comment (Lojjs) [inline modules/ROOT/pages/errors/gql-errors.adoc:110]: This one doesn't seem to be used after all. Maybe I mixed it up with 22N01 when updating the spreadsheet. SO you can comment it out for now.

File: modules/ROOT/pages/errors/gql-errors.adoc [hunk 2]
```diff
@@ -100,6 +100,34 @@ Status description:: error: connection exception - no such routing policy. Polic
...
+=== 22003
+
+Status description:: error: data exception - numeric value out of range. The numeric value { $value } is outside the required range.
```
Comment (Lojjs) [inline modules/ROOT/pages/errors/gql-errors.adoc:113]:
```suggestion
Status description:: error: data exception - numeric value out of range. The numeric value `{ $value }` is outside the required range.
```

To align with the other parameters

**Review by Lojjs — COMMENTED** [review]
I think we must have messed up the spreadsheet somehow for these ones, because it is actually only 22000, 22003 and 22N88 of them which are used.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-201.surge.sh

Updated pages:
- [List of all GQLSTATUS server error codes](https://neo4j-docs-status-codes-201.surge.sh/status-codes/5/errors/gql-errors)

**Comment by NataliaIvakina** [comment]
> I think we must have messed up the spreadsheet somehow for these ones, because it is actually only 22000, 22003 and 22N88 of them which are used.

Ok, then I'll remove them from the PR and comment on the spreadsheet.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-201.surge.sh

Updated pages:
- [List of all GQLSTATUS server error codes](https://neo4j-docs-status-codes-201.surge.sh/status-codes/5/errors/gql-errors)

**Review by Lojjs — APPROVED** [review]
Thanks for fixing, looks good now

**Pull request merged**